server/apps/player/src/character.erl

```diff
@@ -1,179 +1,169 @@
-		   map_name := MapName,
-		   kind := Kind,
-		   x_position := XPosition,
-		   y_position := YPosition},
-		 Connection) ->
-    logger:info("NAME: ~p USERNAME: ~p~n", [Name, Username]),
-    Harvest = "CALL map.harvest_resource($1::TEXT, $2::\"map\".OBJECT_TYPE, $3::REAL, $4::REAL, $5::TEXT, $6::TEXT, $7::TEXT);",
-
-    Inventory = "SELECT item_name, quantity FROM map.resource_item_view WHERE\
-                  name = $1::TEXT AND username = $2::TEXT AND email = $3::TEXT\
-                 LIMIT 1",
-    Resource = "SELECT quantity FROM map.resource WHERE map_name = $1::TEXT AND x_position = $2::REAL AND y_position = $3::REAL AND kind = $4::\"map\".OBJECT_TYPE LIMIT 1",
+                   username := Username,
+                   email := Email,
+                   map_name := MapName,
+                   kind := Kind,
+                   x_position := XPosition,
+                   y_position := YPosition},
+                 Connection) ->
+    logger:debug("[~p] NAME: ~p x USERNAME: ~p~n", [?MODULE, Name, Username]),
+    Harvest = database_queries:fetch_query("map", "harvest_resource.sql"),
+    Inventory = database_queries:fetch_query("character", "select_inventory.sql"),
+    Resource = database_queries:fetch_query("map", "select_resource_quantity.sql"),
     epgsql:with_transaction(Connection,
-				      fun (Conn) ->
-                         do([postgres_m ||
-				_ <- {epgsql:equery(Conn, Harvest, [MapName, Kind, XPosition, YPosition, Name, Email, Username]), call},
-				UnprocessedDeltaInventory <- {epgsql:equery(Conn, Inventory, [Name, Username, Email]), select},
-				UnprocessedDeltaResource <- {epgsql:equery(Conn, Resource, [MapName, XPosition, YPosition, Kind]), select},
-				DeltaInventory = hd(database_utils:columns_and_rows(UnprocessedDeltaInventory)),
-				DeltaResource = case database_utils:columns_and_rows(UnprocessedDeltaResource) of
-                         [A] -> A;
-                         [] -> #{quantity => 0};
-                         _ -> fail("Logical contradiction.") end,
-				return(#{delta_inventory => DeltaInventory, delta_resource => maps:get(quantity, DeltaResource)})])
-				      end,
-			    #{ begin_opts => "ISOLATION LEVEL READ UNCOMMITTED"}).
+                            fun(Conn) ->
+                               do([postgres_m
+                                   || _
+                                          <- {epgsql:equery(Conn,
+                                                            Harvest,
+                                                            [MapName,
+                                                             Kind,
+                                                             XPosition,
+                                                             YPosition,
+                                                             Name,
+                                                             Email,
+                                                             Username]),
```

Comment:
Where are we getting this indentation style? Things are getting uglier. I don't know if not having long lines is beneficial in this case; I feel like the previous way was more readable.